image/config.go

```diff
@@ -73,39 +73,43 @@ func (c *config) runtimeSpec(rootfs string) (*specs.Spec, error) {
 	var s specs.Spec
 	s.Version = specs.Version
 	// we should at least apply the default spec, otherwise this is totally useless
-	s.Process.Terminal = true
-	s.Root.Path = rootfs
-	s.Process.Cwd = "/"
-	if c.Config.WorkingDir != "" {
-		s.Process.Cwd = c.Config.WorkingDir
+	if s.Root != nil {
```

Comment:
I think this is not a right fix. Default Root is nil in Spec, we should initialize it as s.Root= &specs.Root{},  also same to Process.